src/tasks/batch_integration/methods/scanvi/script.py

```diff
@@ -1,30 +1,74 @@
-import yaml
 import anndata as ad
-from scib.integration import scanvi
+from scvi.model import SCVI, SCANVI
 
 ## VIASH START
 par = {
-    'input': 'resources_test/batch_integration/pancreas/unintegrated.h5ad',
+    'input': 'resources_test/batch_integration/pancreas/dataset.h5ad',
     'output': 'output.h5ad',
-    'hvg': True,
+    'n_hvg': 2000,
+    'n_latent': 30,
+    'n_hidden': 128,
+    'n_layers': 2,
+    'max_epochs_scvi': 400,
+    'max_epochs_scanvi': 400
```

Comment:
`max_epochs_scanvi` is different here compare to the value in `src/tasks/batch_integration/methods/scanvi/config.vsh.yaml`. i'd recommend setting both to 20

Reply:
Good point. Fyi -- the values between `## VIASH START` and `## VIASH END` are removed at build time, but it makes sense to keep them aligned.